parsec/core/gui/workspace_button.py

```diff
@@ -376,14 +387,14 @@ def reload_workspace_name(self, workspace_name):
         )
         self.label_shared_info.setToolTip(shared_message)
 
-    def mousePressEvent(self, event):
+    def mousePressEvent(self, event: QMouseEvent) -> None:
         if event.button() & Qt.LeftButton and self.switch_button.isChecked():
             self.clicked.emit(self.workspace_fs)
 
-    def _on_switch_clicked(self, state):
+    def _on_switch_clicked(self, state: Any) -> None:
```

Comment:
`Any` does not looks good, to be generic, `Object` is better than `Any`. Here it appears that state is always a boolean